cafebabel/tests/test_articles.py

```diff
@@ -359,4 +359,5 @@ def test_article_detail_contains_tags(client, app, tag, published_article):
     response = client.get(
         f'/article/{published_article.slug}-{published_article.id}/')
     assert response.status_code == HTTPStatus.OK
-    assert 'Wonderful, Sensational.' in response
+    assert '<a href=/article/tag/wonderful/>Wonderful</a>,' in response
+    assert '<a href=/article/tag/sensational/>Sensational</a>.' in response
```

Comment:
Wondering if the comma end the final point should be written in the test as the HTML may change into a list or a different display.

Reply:
I kept it because it tests conditioning within templates. It's OK to me to change tests when you change the frontend.

Reply:
I see it like the punctuations are not a condition for the test to be valid or not and it may be disturbing for the frontend to have the test broken while not feeling like breaking something.
No big deal anyways, let's keep it then.